sdcm/utils/docker_utils.py

```diff
@@ -384,6 +384,20 @@ def get_container_name_by_id(cls, c_id: str, docker_client: DockerClient = None)
         docker_client = docker_client or cls.default_docker_client
         return docker_client.containers.get(c_id).name
 
+    @classmethod
+    def pause_container(cls, instance: object, name: str):
```

Comment:
Need to add type hint for return value (`-> None:`)

Reply:
fixed